Jenkinsfile

```diff
@@ -0,0 +1,26 @@
+pipeline {
```

Comment:
Do you want this file? We don't use it for open source. If so, can you add SPDX headers

Reply:
I can remove it. But it could be helpful for people using Jenkins.